rules/linux/auditd/lnx_auditd_hidden_binary_execution.yml

```diff
@@ -0,0 +1,27 @@
+title: Hidden Binary Execution
+id: 9e1bef8d-0fff-46f6-8465-9aa54e128c1e
+status: experimental
+description: Hidden files and folders in NIX systems start with ".", it is uncommon for a binary to execute out of a folder that is hidden or for the binary itself to be hidden as this is a common way threat actors attempt to hide.
+author: David Burkett, @signalblur
+references:
+  - https://linux.die.net/man/1/tree
+date: 2022/06/07
+modified: 2022/06/07
+logsource:
+  product: linux
+  service: auditd
+detection:
+  selection:
+    type: 'PATH'
+    name|contains: '/.'
+  filter:
+    name|contains:
+      - '/root/.cache/'
+      - '/.pyenv/'
+  condition: selection and not filter
```

Comment:
@david-burkett did you see the filter that I've added? Is there more that could be excluded right away?

Reply:
@Neo23x0 After reading the other commenter, I think it may make sense to drop it to medium. I've added also a few other non internal ones.